pkg/cmd/target.go

```diff
@@ -739,6 +770,140 @@ func gardenWrapper(targetReader TargetReader, targetWriter TargetWriter, configR
 	return nil
 }
 
+func serverWrapper(reader ConfigReader, serverName string, kubeconfigReader KubeconfigReader) {
+	config := reader.ReadConfig(pathGardenConfig)
+
+	for _, garden := range config.GardenClusters {
+		kc := garden.KubeConfig
```

Comment:
Add a check to see if file path exists if not skip and go on with next
```go
if _, err := os.Stat(kc); os.IsNotExist(err) {
			break
		}
```

Reply:
thanks @DockToFuture , i've fixed this

Reply:
break or continue?

Reply:
`continue` i mean, thanks @petersutter  for checking

Reply:
thanks both, i've fixed and squash into one commit